sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_polling.py

```diff
@@ -64,24 +64,6 @@ def details(self) -> TranslationStatus:
             )
         return TranslationStatus(id=self._polling_method._get_id_from_headers())  # type: ignore # pylint: disable=protected-access
 
-    @classmethod
-    def from_continuation_token(  # type: ignore
-        cls,
-        polling_method: "DocumentTranslationLROPollingMethod",
-        continuation_token: str,
-        **kwargs: Any
-    ) -> "DocumentTranslationLROPoller":  # type: ignore
-        """
-        :meta private:
-        """
-        (
-            client,
-            initial_response,
-            deserialization_callback,
-        ) = polling_method.from_continuation_token(continuation_token, **kwargs)
-
-        return cls(client, initial_response, deserialization_callback, polling_method)
-
```

Comment:
Had this method to appease mypy but impl is same as azure-core. We cast the return type now to override generated code so this is no longer needed.